### PR TITLE
Add repos for Morphy

### PR DESCRIPTION
**Pull request by Fryguy**
Additionally,
- Add ability to release a branch from a different source branch

@chessbyte Please review

### CODE REVIEW

File: bin/release_branch.rb
Comment (Fryguy) [inline bin/release_branch.rb:15]: This bit is a technically bug fix - previously when cutting a branch it would always use "master", even though you're supposed to create a new section.  Previously we always created the repo set as a copy of master, so it worked out, but this time, we created from lasker, so it's different, and it broke.

File: config/repos.yml
Comment (Fryguy) [inline config/repos.yml:116]: @agrare I intentionally didn't include ibm_power_vc and oracle_cloud, so you'll have to update this when those branches are ready

Comment (agrare) [reply config/repos.yml:116]: :+1: perfect

**Comment by Fryguy** [comment]
All branches have been created with this script, with the only issue being something in manageiq-rpm_build.  i don't quite understand the problem yet - cc @bdunne

**Review by chessbyte — APPROVED** [review]
Awaiting green light regarding `manageiq-rpm_build` repo before merging

**Comment by Fryguy** [comment]
This can be merged ahead of time - it's something with the manageiq-rpm_build repo itself, not this PR.  I can run it later against just that one repo when I figure it out.

**Comment by Fryguy** [comment]
Turns out I misunderstood manageiq-rpm_build, and it's fine.

**Pull request merged**
